### ISSUE TITLE
Support Request: 

### ISSUE THREAD

**Issue by CorporateASH** [bloomkd46/homebridge-XfinityHome#13]
### Describe Your Problem:

I'm installing this Homebridge plugin on a Synology DS218 NAS. Homebridge has loaded correctly and is running one other plugin (LIFX). This plugin appears to install correctly, however it doesn't seem to operate. The configuration UI just presents a spinning cog. This is a fresh install of Homebridge two hours before installing this plugin. Is there something I need to alter in order to use this plugin?

Thanks in advance.

### Logs:

```shell
[1/7/2023, 9:59:45 PM] [Homebridge UI] [homebridge-xfinityhome] /volume1/homebridge/node_modules/homebridge-xfinityhome/homebridge-ui/server.js:30
            (((_c = config._bridge) === null || _c === void 0 ? void 0 : _c.username) ? ('.' + ((_e = (_d = config._bridge) === null || _d === void 0 ? void 0 : _d.username) === null || _e === void 0 ? void 0 : _e.split(':').join(''))) : '');
                           ^

TypeError: Cannot read properties of undefined (reading '_bridge')
    at new PluginUiServer (/volume1/homebridge/node_modules/homebridge-xfinityhome/homebridge-ui/server.js:30:28)
    at /volume1/homebridge/node_modules/homebridge-xfinityhome/homebridge-ui/server.js:270:8
    at Object.<anonymous> (/volume1/homebridge/node_modules/homebridge-xfinityhome/homebridge-ui/server.js:270:29)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
[1/7/2023, 9:59:45 PM] [Homebridge UI] [homebridge-xfinityhome] Child process ended
[1/7/2023, 9:59:47 PM] [Homebridge UI] [homebridge-xfinityhome] Terminating child process...
```

### Config:

```shell
{
    "bridge": {
        "name": "Homebridge CCFB",
        "username": "0E:DA:B1:78:CC:FB",
        "port": 51775,
        "pin": "392-19-053",
        "advertiser": "ciao",
        "bind": [
            "eth0"
        ]
    },
    "accessories": [],
    "platforms": [
        {
            "name": "Config",
            "port": 8581,
            "auth": "form",
            "theme": "auto",
            "tempUnits": "f",
            "lang": "auto",
            "platform": "config"
        },
        {
            "name": "Lifx Plugin",
            "duration": 0,
            "brightnessDuration": 300,
            "colorDuration": 300,
            "default": "0.0.0.0",
            "broadcast": "255.255.255.255",
            "lightOfflineTolerance": 3,
            "messageHandlerTimeout": 45000,
            "resendPacketDelay": 150,
            "resendMaxTimes": 3,
            "updates": true,
            "debug": false,
            "autoDiscover": true,
            "platform": "LifxPlugin"
        }
    ]
}
```

### Operating System:

Synology OS

### Node Version:

16.16.0

### NPM Version:

8.11.0

### Homebridge Version:

1.6.0

### Plugin Version:

3.0.4

**Comment by BSCTech70** [comment]
I am having the same issue, and it won't finalize the installation and go into the configuration. The gear continues to spin.

**Comment by patricksmith0330** [comment]
I had the same issue, I installed an older version and was able to get it configured then update to the latest. Working perfectly ever since.

**Comment by bloomkd46** [comment]
Fixed in 3.0.5
